### DISCUSSION TITLE
Help needed - Export Mindmaps: Image formats - Jpeg, png, svg

### DISCUSSION THREAD

**Discussion by scm1599** [freeplane/freeplane#2021] [Help]
FreeplanePortable-1.12.5.paf.exe
Topic:
Export Mindmaps: Image formats - Jpeg, png, svg
According to manual Jpeg, png, svg are possible
https://docs.freeplane.org/user-documentation/working-w-mindmap-files.html?highlight=svg#export
However:
png, svg: not displayed in the menu
Jpeg is displayed in menu, but runs in an error, failed export

What I really need is Export Mindmaps to svg.

Any ideas?

**Comment by euu2021** [comment]
Png doesn't show in your dropdown menu, like this?

![image](https://github.com/user-attachments/assets/80337780-7717-466a-8f63-61be71d1c90e)

Error exporting may be related to this: https://github.com/freeplane/freeplane/issues/1356

**Reply by scm1599** [reply]
Png shows up as in the screenshot, but does not work. Error Message appears.
More severe:
SVG does not show up in the dropdown menu at all.

**Comment by dpolivaev** [comment]
If you see any errors and need support from me, open a bug report issue and upload your log files containing the error traces.

**Comment by scm1599** [comment]
Thanks, What about the SVG export? According to the manual there is SVG export. However, there is no SVG export in the menu. Is this an error, or is there no SVG export at all? Actually there must be SVG export. SVG export is also mentioned in the history, not only in the manual.
PNG is a mistake, I will make a report.

**Reply by dpolivaev** [reply]
Not having the SVG export entry looks like an error. If you need support from me, open a bug report issue and upload your log files.

**Comment by scm1599** [comment]
Tested again: SVG is there. I had overlooked the scrollbar in the menu.
**JPG does not work, runs on error,** definitely not working. I will make a bug report.

**Comment by scm1599** [comment]
JPG Export Bug Report opened:
https://github.com/freeplane/freeplane/issues/2022
